src/D2L.Security.OAuth2/Keys/Default/RsaPrivateKeyProvider.cs

```diff
@@ -14,6 +15,7 @@ ID2LSecurityTokenFactory d2lSecurityTokenFactory
 			m_d2lSecurityTokenFactory = d2lSecurityTokenFactory;
 		}
 
+		[GenerateSync]
 		Task<D2LSecurityToken> IPrivateKeyProvider.GetSigningCredentialsAsync() {

```

Comment:
For not-actually-async functions like this; I wonder if we'd ultimately want to do something different?

Reply:
@j3parker

Reply:
I understand that, yes.